auth/src/main/res/layout-land/fui_auth_method_picker_layout.xml

```diff
@@ -22,6 +22,7 @@
         app:layout_constraintEnd_toStartOf="@+id/container"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintBottom_toTopOf="@+id/main_tos_and_pp"
+        app:layout_constraintVertical_weight="2"
```

Comment:
This works, but I was also thinking what we might want is to center the logo between the top of the parent and the top of the ToS/PP.  What do you think about that?

Reply:
I think it's already doing that, right? At least, that's what I'm getting from this screenshot. The issue seems to be that there's a lot more padding than we're used to.
![screenshot_20180924-165834](https://user-images.githubusercontent.com/9490724/45985445-2576e900-c01b-11e8-9bc0-a64a39c70038.png)